README.md

```diff
@@ -16,15 +16,17 @@ To get the VMs up and running, you need some things installed on your developmen
+| Name                                | OS              | Description                                                                                                                                                                                                              | Installed Contents                                                                                                                                  |
+| ----                                | --              | -----------                                                                                                                                                                                                              | --------                                                                                                                                            |
+| ubuntu-trusty-rust_slave            | Ubuntu 14.04    | Use for building Crust on Ubuntu. This version of Ubuntu matches the version in use on Travis, so it should be very similar to their environment.                                                                        | gcc, libssl-dev, pkg-config, musl-dev, musl-tools, rust 1.29.2, x86_64-unknown-linux-musl target, cargo-script                                      |
+| centos-7-rust_slave                 | CentOS 7.5      | Use for building Crust on CentOS. Currently it won't build due to a weird issue with openssl.                                                                                                                            | gcc, openssl-devel, musl compiled from source (no packages on available on CentOS), rust 1.29.2, x86_64-unknown-linux-musl target, cargo-script     |
+| windows-2012_r2-base                | Windows 2012 R2 | Use as a base box when you need a completely clean machine.                                                                                                                                                              | Only the base operating system.                                                                                                                     |
+| windows-2012_r2-msys2-rust_slave    | Windows 2012 R2 | Use for building Crust on Windows. This is setup to use MinGW rather than MSVC.                                                                                                                                          | MSYS2, mingw-w64-x86_64-gcc, rust 1.29.2, ConEmu, Chrome (the last 2 are just for alternatives to the unfortunately poor default tools on Windows). |
+| windows-2012_r2-git_bash-rust_slave | Windows 2012 R2 | Use for debugging issues running Windows builds on Travis, which uses Git Bash as a shell. The reason why it's a separate box is because sometimes Git Bash and MSYS2 can interfere with each other. | Git Bash, rust 1.29.2, ConEmu, Chrome (the last 2 are just for alternatives to the unfortunately poor default tools on Windows). |
```

Comment:
Is/Should the idea in the table be that column 1 is the exact name you use in your make command? I.e. the value in column 1 matches the machine name in the Makefile? There's a slight difference with the 2 non-base Windows machines as the table has a `-` after the word `windows`, while the Makefile doesn't

Reply:
Thanks for that! I've now made the machine/target names consistent as it should be.